test/libsolidity/syntaxTests/bytecodeReferences/library_non_called.sol

```diff
@@ -7,5 +7,7 @@ library L2 {
 contract A {
     function f() public pure { type(L2).creationCode; }
```

Comment:
Should `creationCode` and `runtimeCode` be unavailable for EOF? They brake code non observability rule, right? Assert is also needed, but it should be also changed in higher level to prevent usage of them in a contract code.